pkg/reconciler/v1alpha1/configuration/resources/revision.go

```diff
@@ -41,6 +41,9 @@ func MakeRevision(config *v1alpha1.Configuration) *v1alpha1.Revision {
 	}
 	rev.Labels[serving.ConfigurationLabelKey] = config.Name
 
+	configLabels := config.Labels
+	rev.Labels[serving.ServiceLabelKey] = configLabels[serving.ServiceLabelKey]
```

Comment:
Can you add an issue to update our conformance tests to track the set of labels propagated between Service -> Configuration -> Revision?

Reply:
created issue #2021